sdk/identity/identity/test/manual-integration/AzureVM/INSTRUCTIONS.md

```diff
@@ -0,0 +1,126 @@
+# prerequisite tools
+- Azure CLI
+
+# Azure resources
```

Comment:
I believe the PR looks fine, but I lack the context to understand who might need to run these tests. Are we expecting users to see this bundled with identity? Is this only for developers? Feels like there should be a way to connect the rest of this repository with this content.

Here's one scenario that I'm interpreting from this:
- Developers might need to run these tests while they are altering the identity package.

In that scenario, would a developer from another client check out the README at the root of identity and know that they have to enter the manual-integration folder and run the tests?

I also wonder if we should have guidelines (eventually, at least) for these kind of tests.

Anyway, I'm adding this comment this place so that you can answer me as a thread. This is just out of curiosity. Please let me know if what I'm describing doesn't provide any value for this PR! That's a valid response too 🌞 thank you for your time!

Reply:
Users won't use these, though they can look at them as examples of how to interact with the various services. The intent is to have a script and test files that someone could use to test by hand. They'll mostly intended to be used as part of the manual verification process of the SDK.

Developers shouldn't need to update the tests unless the API changes.

We don't have any official guidelines, but that is something we could look into in the future. I'm also not sure how much we'll automate these in the future. We're still early in the design process.